lib/vdsm/tool/config_lvm_filter.py

```diff
@@ -218,9 +227,20 @@ def _print_filter_warning():
     """)
 
 
+def _print_devices_info(vgs):
+    print("""\
+Configuring LVM to use devices file instead of LVM filter.
+Devices for following VGs will be imported:
+    """)
+    print("  ", ", ".join(vgs))
```

Comment:
This will print 3 spaces before the vgs list - is this intended?

To have better control and make the code more clear, you can use:

    print(f"  {', '.join(vgs)}")